tests/src/Rector/Deprecation/UiHelperTraitDrupalPostFormRector/UiHelperTraitDrupalPostFormRectorTest.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace DrupalRector\Tests\Rector\Deprecation\UiHelperTraitDrupalPostFormRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+/**
+ * @coversDefaultClass \DrupalRector\Rector\Deprecation\DatetimeStorageTimezoneRector
+ */
+class UiHelperTraitDrupalPostFormRectorTest extends AbstractRectorTestCase {
+
+    /**
+     * @covers ::refactor
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        $this->markTestIncomplete(
+            'Fails since BrowserTestBase cannot be resolved, will work when drupal/core added as a dev dependency.'
+        );
+        $this->doTestFileInfo($fileInfo);
```

Comment:
We don't need the dev dependency. We can add a stub file, like in https://github.com/rectorphp/rector-symfony/tree/main/stubs